api/api-doc/storage_service.json

```diff
@@ -1061,6 +1061,21 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/reset_cleanup_needed",
```

Comment:
The fact that there is some "cleanup needed" flag seems to be an internal detail. Do we really need to include it in the name and the summary?

Reply:
I do not think we need expose vnode detail to users, tablets are not affected by it in any way. @gusev-p changes was internal to the code. I agree though that `reset_cleanup_needed` expose internal implementation as well. May be `mark_node_as_clean`.